api/client/build.go

```diff
@@ -51,7 +52,7 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 	cmd := Cli.Subcmd("build", []string{"PATH | URL | -"}, Cli.DockerCommands["build"].Description, true)
 	flTags := opts.NewListOpts(validateTag)
 	cmd.Var(&flTags, []string{"t", "-tag"}, "Name and optionally a tag in the 'name:tag' format")
-	suppressOutput := cmd.Bool([]string{"q", "-quiet"}, false, "Suppress the verbose output generated by the containers")
+	suppressOutput := cmd.Bool([]string{"q", "-quiet"}, false, "Suppress verbose output and print image ID, if succeed")
```

Comment:
Perhaps s/verbose/build/  since its not just chatty/verbose output, its all of the build output.
But up to you, not a big deal either way.